pkgs/development/libraries/libschrift/default.nix

```diff
@@ -2,18 +2,22 @@
 
 stdenv.mkDerivation rec {
   pname = "libschrift";
-  version = "0.9.1";
+  version = "0.10.0";
 
   src = fetchFromGitHub {
     owner = "tomolt";
     repo = pname;
-    rev = "c6d20460d6e602e8829d3a227fd7be4c4c3cda86";
-    hash = "sha256-BuTmWaWFZ0DXujlbhbmK3Woit8fR9F4DWmKszHX6gOI=";
+    rev = "c207585486b3e78ec5506f55f5d56178f421a53c";
+    sha256 = "13qrplsi2a53s84giwnzqmn0zbslyaagvjn89wsn9fd90m2v2bs1";
   };
 
   postPatch = ''
     substituteInPlace config.mk \
       --replace "PREFIX = /usr/local" "PREFIX = $out"
+  '' + lib.optionalString stdenv.isDarwin ''
+    # on darwin unsigned long int != uint_fast32_t
+    sed -i 's/typedef unsigned long\s\+SFT_Glyph;/#include <stdint.h>\
```

Comment:
Should we use a patch for this? This could easily bit rot.

Reply:
Now that the same change has been merged into upstream I could probably add the commit as a patch, but it's gonna have to be removed at the next release either way.